tracer/test/snapshots/DataStreamsMonitoringTests.SubmitsDataStreams.verified.txt

```diff
@@ -141,6 +141,143 @@
           PayloadSize: /w==,
           TimestampType: current
         }
+      ],
+      Backlogs: [
+        {
+          Tags: [
```

Comment:
I could be missing something, but shouldn't we have offsets in here too? 🤔

Reply:
They are all set to 0, which is excluded during serialization. I reset the offsets, since there's no guarantee messages will consistently land on the same partition each time we run the test.

Reply:
Ah, cool, makes sense, thanks!